src/components/fx/hover.js

```diff
@@ -577,9 +579,17 @@ function createHoverText(hoverData, opts) {
             ltext = label.selectAll('text').data([0]);
 
         lpath.enter().append('path')
-            .style({fill: Color.defaultLine, 'stroke-width': '1px', stroke: Color.background});
+            .style({
+                fill: commonLabelOpts.bgcolor || Color.defaultLine,
+                stroke: commonLabelOpts.bordercolor || Color.background,
```

Comment:
After some thought, I think it would be best to **not** render the alpha channel in all hover labels, as the marker hover labels already [blend](https://github.com/plotly/plotly.js/pull/1582/files#diff-53eaf7f3ac3274b40206f89f8f87a5d6R691) marker opacities with `plot_bgcolor` filtering out the alpha channel. For example,

```js
var trace = {
  x: [1],
  y: [1],
  marker: {
    color: ['rgba(255, 0, 0, 0.3)']
  }
}

var layout = {
  plot_bgcolor: 'grey'
}
```

renders the marker hover label as `rgb(166, 90, 90)` i.e. with no alpha channel.

We could add some special logic so that e.g.

```js
var trace = {
  x: [1],
  y: [1],
  marker: {
    color: ['rgba(255, 0, 0, 0.3)']
  },
  hovelabel: {
    bgcolor: 'rgba(0,0,255,0.3)'
  }
}

var layout = {
  plot_bgcolor: 'grey'
}
```

renders the alpha channel overriding the marker color to hover label color behavior, but quite frankly I don't see the need.

Adding a line in the schema description saying that alpha channel aren't honored sounds sufficient for now. Thoughts?